src/angular/button/button.spec.ts

```diff
@@ -684,6 +684,9 @@ class ButtonIconTest extends ButtonTestBase {}
       <a sbb-alt-button type="button"><sbb-icon svgIcon="example"></sbb-icon></a>
       <a sbb-secondary-button type="button"><sbb-icon svgIcon="example"></sbb-icon></a>
       <a sbb-ghost-button type="button"><sbb-icon svgIcon="example"></sbb-icon></a>
+      <button sbb-button type="button">
```

Comment:
Is this valid HTML?

Reply:
Yes, but makes no sense. I removed it.

Reply:
No, you're right. I didn't know this.
![image](https://user-images.githubusercontent.com/3184240/190962334-fe0d875f-f107-41c7-a3a1-dfee1cebc96b.png)

This bug appeared in our Tekton-UI which is why I created this PR. I close it and write a comment there.

Reply:
I was vaguely aware that there was some restriction and wanted to be sure.
Thanks for checking! 👍